org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4whole/SimpleGUI.java

```diff
@@ -1734,7 +1740,10 @@ public String compute(Object input) {
 			SimpleTask2 task = new SimpleTask2();
 			task.filename = arg;
 			try {
-				WorkerEngine.run(task, SubMemory.get(), SubStack.get(), alloyHome() + fs + "binary", "", cb);
+				if ("yes".equals(System.getProperty("debug")))
```

Comment:
Don't copy-paste: there is this exact same expression on line 1852 (and possibly elsewhere).  Let's have a helper function and reuse it at least everywhere in this file.

Reply:
Again, all to try to minimize changes. Problem is that Alloy has no Context object. In this case we can make it a static method somewhere (in core I assume). I added AlloyCore in the core project as a placeholder and added an isDebug.

However, there are a million of these chores in the code ...